Examples/Complete/Materials/Core/Materials.cs

```diff
@@ -398,38 +396,38 @@ public override void Init()
                                     icosphereWithTangents
                                 }
                             },
-                            new SceneNode
-                            {
-                                Components = new List<SceneComponent>
-                                {
-                                    new Transform
-                                    {
-                                        Name = "specular texture",
-                                        Translation = new float3(0, 0, 0)
-                                    },
-                                    ShaderCodeBuilder.MakeShaderEffectFromShaderEffectPropsProto(new ShaderEffectProps
-                                    {
-                                        MatProbs =
-                                        {
-                                            HasAlbedo = true,
-                                            HasAlbedoTexture = true,
-                                            HasSpecular = true,
-                                            HasSpecularTexture = true
-                                        },
-                                        MatType = MaterialType.Standard,
-                                        MatValues =
-                                        {
-                                            AlbedoColor = new float4(0.39f, 0.19f, 0, 1),
-                                            SpecularColor = float4.One,
-                                            SpecularIntensity = 2f,
-                                            SpecularShininess = 25f,
-                                            SpecularMix = 1f, // TODO: Implement in ShaderShards
-                                            SpecularTexture = "specularTex.jpg" // TODO: Implement in ShaderShards
-                                        }
-                                    }),
-                                    icosphereWithTangents
-                                }
-                            },
+                            //new SceneNode
```

Comment:
Perhaps we could add a reason why we leave commented code and why it's commented out in the first place. As far as I recon, it's due to the fact, that there is no `SpecularTexture` in ShaderShards?

Reply:
Not sure, but I agree with commenting commented out code.

Reply:
Yes, this is due to the fact that specular textures are not implemented yet. We do not have shader code that is capable of using them. This lead to strange results where the specular texture was interpreted as albedo texture.